opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -536,9 +536,9 @@ message ExponentialHistogramDataPoint {
     // Note: This uses a varint encoding as a simple form of compression.
     sint32 offset = 1;
 
-    // Count is an array of counts, where count[i] carries the count
-    // of the bucket at index (offset+i).  count[i] is the count of
-    // values greater than base^(offset+i) and less or equal to than
+    // BucketCounts is an array of count values, where BucketCounts[i] carries
+    // the count of the bucket at index (offset+i). BucketCounts[i] is the count
+    // of values greater than base^(offset+i) and less than or equal to
```

Comment:
Not blocking this PR, do we feel this might still be confusing since there are positive AND **negative** buckets?

Reply:
I am going to merge the PR, please open an issue if you feel it is necessary.